no-computer.md

```diff
@@ -1,69 +1,170 @@
 # No computer project
 
 
+
+Blueprints for a building. Pretty straightforward.
+
+
+
 ## The user and a language
+
 This section describes who the project would serve and why a language might be a
+
 good way to meet their needs.
 
 
+
+This would serve architectual engineers, construction workers, and other people who would need to know exactly what to do to to create the building.
+
+
+
 ### What's the need?
+
 _What need is met by your idea? Who are you helping? What is that person's
+
 experience like now? What would their experience be like if you could help 
+
 them?_
 
 
+
+Everyone listed above. Without blueprints, it may be possible to create buildings, but not skyscrapers. Anything that takes into account the laws of physics would probably need detailed planning.
+
+
+
 ### Why a language?
+
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
 
+
+It lets them order a more accurate amount of materials needed and plan out the order to build things in.
```

Comment:
Hmm, but why a _language_? What about the information in  blueprints says language? What about the material content? What about the way in which they're used?